glide.yaml

```diff
@@ -79,3 +79,4 @@ import:
 - package: github.com/jaytaylor/html2text
 - package: gopkg.in/gomail.v2
   version: 2.0.0
+- package: github.com/ssor/bom
```

Comment:
@cpanato I haven't heard a response yet about adding an open-source license -- while we should wait for a few more days, I'm wondering if there are other alternatives we could consider?